app/models/manageiq/providers/redhat/network_manager/refresher.rb

```diff
@@ -0,0 +1,20 @@
+module ManageIQ::Providers
+  class Redhat::NetworkManager::Refresher < Openstack::NetworkManager::Refresher
+    def collect_inventory_for_targets(ems, targets)
+      targets_with_data = targets.collect do |target|
+        target_name = target.try(:name) || target.try(:event_type)
+
+        _log.info("Filtering inventory for #{target.class} [#{target_name}] id: [#{target.id}]...")
+
+        if ::Settings.ems.ems_ovirt.ems_network.try(:refresh).try(:inventory_object_refresh)
+          inventory = ManageIQ::Providers::Redhat::Builder.build_inventory(ems, target)
+        end
+
+        _log.info("Filtering inventory...Complete")
+        [target, inventory]
```

Comment:
What would be the result if line 9 returns false and not value is set to `inventory` ?

Reply:
The refresh of the 'network manager' will fail.